.github/workflows/pipeline.yml

```diff
@@ -23,6 +25,7 @@ jobs:
         env:
           PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: "true"
+          YARN_CHECKSUM_BEHAVIOR: "ignore"
```

Comment:
This shouldn't be necessary (see [here](https://yarnpkg.com/advanced/error-codes#yn0018---cache_checksum_mismatch)).